### ISSUE TITLE
DB/Creature: Rattlecage Skeleton - Battle Shout

### ISSUE THREAD

**Issue by ArcaneFox** [TrinityCore/TrinityCore#21228]
**Description:**
The Battle Shout of the Rattlecage Skeleton applying to the player too.
http://www.wowhead.com/npc=1890/rattlecage-skeleton

**Expected behaviour:**
The Battle Shout of the Rattlecage Skeleton isn't applying to the player.

**Steps to reproduce the problem:**

1. .tele  Deathknell

**Branch(es):** master

**TC rev. hash/commit:** https://github.com/TrinityCore/TrinityCore/commit/3d7af56f5cab503518fb2202ca839793c2b08862

**TDB version:**  720 + Updates

**Operating system:** Windows 10 x64

**Comment by pedroevil51** [comment]
Have you checked the faction id for the creature?

**Comment by tkrokli** [comment]
Good point. Could it be that neutral factions are unable to apply hostile spells to target victims?

**Comment by Killyana** [comment]
You must test it with gm off.

**Comment by Killyana** [comment]
Need to be confirmed in the current revision
